src/map-caption.js

```diff
@@ -0,0 +1,43 @@
+import './leaflet.js'; 
+import './mapml.js'; 
+
+/* 
+implemented for both mapml-viewer and web-map; however web-map does not focus on map element in the browser resulting in NVDA 
+not being able to read out map-caption and stating that it's an interactive map region
+*/
+export class MapCaption extends HTMLElement {
+    constructor() {
+        super();
+        // calls MutationObserver; needed to observe changes to content between <map-caption> tags and update to aria-label
+        this.observer = new MutationObserver(() => {
+            if (document.querySelector('map-caption').innerHTML == this.parentElement.getAttribute('aria-label'))
```

Comment:
Should use === instead of ==
```suggestion
            if (document.querySelector('map-caption').innerHTML === this.parentElement.getAttribute('aria-label'))
```